azure-functions-maven-plugin/src/test/java/com/microsoft/azure/maven/function/invoker/storage/EventHubProcesser.java

```diff
@@ -0,0 +1,152 @@
+        }
+    }
+
+    public EventHub createOrGetEventHubByName(String eventHubName) throws Exception {
+        final Azure azureClient = CommonUtils.getAzureClient();
+        final List<EventHub> resultList = eventHubNamespace.listEventHubs().stream()
+                .filter(eventHub -> eventHub.name().equals(eventHubName)).collect(Collectors.toList());
+        if (resultList.size() == 0) {
+            return azureClient.eventHubs().define(eventHubName).withExistingNamespace(eventHubNamespace)
+                    .withExistingStorageAccountForCapturedData(storageAccount, eventHubName)
+                    .withDataCaptureEnabled().create();
+        } else {
+            return resultList.get(0);
+        }
+    }
+
+    public void sendMessageToEventHub(String eventhub, String message) throws Exception {
+        final EventHubClient ehClient = getEventHubClientByName(eventhub);
+        final String payload = message;
+        final Gson gson = new Gson();
+        final byte[] payloadBytes = gson.toJson(payload).getBytes(Charset.defaultCharset());
+        final EventData sendEvent = EventData.create(payloadBytes);
+        ehClient.send(sendEvent).get();
+    }
+
+    public List<String> getMessageFromEventHub(String eventhub) throws Exception {
+        final List<String> result = new ArrayList<>();
+        final EventHubClient ehClient = getEventHubClientByName(eventhub);
+        final List<String> partitionIds = Arrays.asList(ehClient.getRuntimeInformation().get().getPartitionIds());
+        partitionIds.parallelStream()
+                .forEach(partitionId -> result.addAll(receiveMessageFromSpecificPartition(ehClient, partitionId)));
+        return result;
+    }
+
+    public List<String> receiveMessageFromSpecificPartition(EventHubClient eventHubClient, String partitionId) {
+        final List<String> result = new ArrayList<>();
+        try {
+            final PartitionReceiver partitionReceiver = eventHubClient
+                    .createEpochReceiver(EventHubClient.DEFAULT_CONSUMER_GROUP_NAME,
+                            partitionId, EventPosition.fromStartOfStream(), 1000).get();
+            final Iterable<EventData> data = partitionReceiver.receive(10).get();
+            if (data != null) {
+                data.forEach(eventData -> result.add(new String(eventData.getBytes())));
+            }
+            partitionReceiver.closeSync();
+        } catch (Exception e) {
+            // When exception, just return empty List
+            e.printStackTrace();
+        }
+        return result;
+    }
+
+    public void close() throws Exception {
+        for (final EventHubClient eventHubClient : eventHubClientMap.values()) {
+            eventHubClient.closeSync();
+        }
+        executorService.shutdown();
+    }
+
+    private EventHubClient getEventHubClientByName(String eventHubName) throws Exception {
+        if (!eventHubClientMap.containsKey(eventHubName)) {
```

Comment:
Suggest the other way around:
```Java
if (eventHubClientMap.containsKey(eventHubName)) {
    return eventHubClientMap.get(eventHubName);
}
// handle the key not exists case
```

Reply:
I prefer the origin design for i just do single return in this method, while i will write two return statement in this method if i process contains first.